website/docs/workloads/blender/blender-profiles.md

```diff
@@ -0,0 +1,48 @@
+# Blender Workload Profiles
+The following profile runs the Blender benchmark Workloads.
+
+* [Workload Details](./blender.md)  
+
+## PERF-BLENDER-AMD.json
+Runs the Blender Workloads.
+
+* [Workload Profile](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-BLENDER-AMD.json) 
+
+* **Supported Platform/Architectures**
+  * win-x64
+  * AMD v620 GPU or AMD Mi25 GPU
```

Comment:
Do you know if it's true? I imagine it should work everywhere even without GPU?

Reply:
Yes the workload does produce CPU scores on system without a GPU, but the AMDGPU installation step will fail. Do we consider this as supported?